plugins/ibm_qradar/plugin.spec.yaml

```diff
@@ -0,0 +1,997 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ibm_qradar
+title: IBM QRadar
+vendor: rapid7
+support: community
+status: []
+supported_versions: [7.3.3]
+description: IBM QRadar reduces billions of events and flows into a manageable number of actionable offenses that are prioritized by their impact on your business operations. This plugin allows you to use IBM QRadar to orchestrate and automate Ariel search queries and automate offense management
+version: 1.0.0
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/ibm_qradar
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ibm.com
+tags:
+  - qradar
+  - ariel
+  - aql
+  - offenses
+  - orchestration
+hub_tags:
+  use_cases: [asset_inventory, vulnerability_management, offensive_security]
+  keywords: [siem, search]
+  features: []
+enable_cache: false
+connection:
+  host_url:
+    title: Host URL
+    type: string
+    description: Host URL of the QRadar instance.
+    required: true
+    example: https://127.0.0.1
+  username:
```

Comment:
Please change the definition of the connection username/password credential to use the SDK-built-in base type `credential_username_password`.
[Here](https://docs.rapid7.com/insightconnect/plugin-spec#base-types) is our documentation on base types built into the SDK.
Here is an example of using the username/password credential base type for the connection in the Microsoft Teams plugin, in the [plugin spec](https://github.com/rapid7/insightconnect-plugins/blob/master/plugins/microsoft_teams/plugin.spec.yaml#L266) and in the [connection.py](https://github.com/rapid7/insightconnect-plugins/blob/master/plugins/microsoft_teams/icon_microsoft_teams/connection/connection.py#L18) file.